media-sound/drumkv1/drumkv1-9999.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fdo-mime
+
+DESCRIPTION="An old-school all-digital drum-kit sampler synthesizer with stereo fx"
+HOMEPAGE="http://drumkv1.sourceforge.net/"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/rncbc/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://download.sourceforge.net/${PN}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+LICENSE="GPL-2"
+SLOT="0"
+RESTRICT="mirror"
+
+IUSE="debug -qt4 +standalone alsa +jackmidi jacksession lv2 osc"
+REQUIRED_USE="
+	|| ( standalone lv2 )
+	standalone? ( || ( alsa jackmidi ) )
+	jackmidi? ( standalone )
+	jacksession? ( standalone )"
+
+RDEPEND="
+	qt4? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+	)
+	!qt4? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtxml:5
+	)
+	media-libs/libsndfile
+	standalone? ( virtual/jack )
+	alsa? ( media-libs/alsa-lib )
+	lv2? ( media-libs/lv2 )
+	osc? ( media-libs/liblo )
+"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	if [[ ${PV} == *9999 ]]; then
+		eautoreconf
+	fi
+
+	# Disable stripping
+	echo "QMAKE_STRIP=" >> src/src_core.pri.in
```

Comment:
> I see, but shouldn't you prevent portage from stripping the binaries when using the debug flag ?

No, that's not something the `debug` USE flag should determine

`FEATURE=nostrip` is a portage thing, which you could set globally, once when emerging or per package in portage's `env.d`

`USE=debug` is a package thing that enables/disables creating a debug build for that package (or globally if you'd set it in your `make.conf`)